ci/build_stf.yml

```diff
@@ -0,0 +1,46 @@
+---
+- name: "Do a local build of STF"
+  hosts: controller
+  tasks:
+    - name: "Set the value of sto_dir"
+      ansible.builtin.set_fact:
+        sto_dir: '{{ ansible_env.HOME }}/{{ zuul.project.src_dir }}'
+
+    - name: "Check for scripts"
+      ansible.builtin.stat:
+        path: "{{ sto_dir }}/build"
+      register: stat
+
+    - name: "Show the contents of {{ sto_dir }}"
+      ansible.builtin.debug:
+        var: stat
+
+    - name: "Get registry info"
+      ansible.builtin.shell:
+        cmd: |
+          oc registry info --internal
+      register: registry
+
+    - name: "Show the registry information"
+      ansible.builtin.debug:
+        var: registry.stdout
+
+    - name: "Build STF containers"
+      ansible.builtin.import_role:
+        name: '../build/stf-run-ci'
+      vars:
+        __deploy_stf: true
+        __local_build_enabled: true
+        __deploy_from_index_enabled: true
+          #__deploy_from_bundles_enables: false
+        __service_telemetry_snmptraps_enabled: true
+        __service_telemetry_storage_ephemeral_enabled: true
+        __working_branch: master
```

Comment:
Should this be the current branch instead of the master branch? Can be found with `git rev-parse --abbrev-ref HEAD`

Reply:
Yes it should, however, this var is used for the cloning stage, which I plan to skip with zuul because zuul pre-clones and checks out the right changes.

Reply:
Sorry, I don't think I understand the answer here. Not sure how this is being skipped, or why you are supplying `__working_branch: master` in this case.

Reply:
It's a work-in-progress. stf-run-ci does its own cloning with the clone_repos task.

With zuul, we expect the correct branches to be checked out for each project.

Adding an option to skip cloning, set my own values of <repo>_dir for each component is covered in STF-1746

__working_branch needs to be removed from here. It is not needed when cloning is skipped (which I notice I've done on this branch)